pkgs/development/interpreters/ruby/bundler-head.nix

```diff
@@ -1,13 +1,9 @@
 { buildRubyGem, coreutils, fetchgit }:
 
 buildRubyGem {
-  name = "bundler-2015-01-11";
-  src = fetchgit {
-    url = "https://github.com/bundler/bundler.git";
-    rev = "a2343c9eabf5403d8ffcbca4dea33d18a60fc157";
-    sha256 = "1p7kzhmicfljy9n7nq3qh6lvrsckiq76ddypf6s55gfh1l98z4k9";
-    leaveDotGit = true;
-  };
+  name = "bundler-1.8.9";
+  namePrefix = "";
+  sha256 = "1k4sk4vf0mascqnahdnqymhr86dqj92bddciz5b2p9sv3qzryq57";
```

Comment:
yep, this works the same way as https://github.com/NixOS/nixpkgs/blob/release-15.09/pkgs/development/interpreters/ruby/bundler.nix